x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/init_and_status_apis.ts

```diff
@@ -27,7 +27,7 @@ export default ({ getService }: FtrProviderContext) => {
   const riskEngineRoutes = riskEngineRouteHelpersFactory(supertest);
   const log = getService('log');
 
-  describe('@ess @serverless init_and_status_apis', () => {
```

Comment:
We have 2 open issue related to failures on this test:
https://github.com/elastic/kibana/issues/179578 https://github.com/elastic/kibana/issues/182541

Reply:
Wonderful callout @machadoum !

I saw you have a PR that hopefully addresses the issue, that's great to see. Have we already run the flaky test runner against your PR?

Reply:
Yes, 25 times here: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/5978